### PR TITLE
Conversacion enano

### PR DESCRIPTION
**Pull request by aaviless**
"Please review this, @casey-collab

### CODE REVIEW

File: espanol/rezando/enano/enano.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+Acto seguido el enano golpeó detrás de las piernas a Jurema, que calló al suelo de rodillas.
 Sin permitir hablar a Jurema el enano habló: "Soy Tuleto y he venido a iluminar sus mentes con la luz del ..."
 
-[Amo del fuego](fuego/fuego.md)
\ No newline at end of file
+[Amo del fuego](fuego/fuego.md)
+[Ombligo de Cacatucán](ombligo/ombligo.md)
```
Comment (casey-collab) [inline espanol/rezando/enano/enano.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by aaviless** [comment]
The blank line is fixed! Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :melon:

**Pull request merged**